api/proto/relay/relay.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package node;
+import "google/protobuf/wrappers.proto";
+import "common/common.proto";
+option go_package = "github.com/Layr-Labs/eigenda/api/grpc/relay";
+
+/////////////////////////////////////////////////////////////////////////////////////
+// Experimental: the following definitions are experimental and subject to change. //
+/////////////////////////////////////////////////////////////////////////////////////
+
+service Relay {
+  // Note: the relay API is still being fleshed out. This definition is not yet complete.
```

Comment:
nit: align the indent style with other .proto files

Reply:
done